autopilot/simple_graph.go

```diff
@@ -62,5 +69,156 @@ func NewSimpleGraph(g ChannelGraph) (*SimpleGraph, error) {
 		graph.Nodes[nodeIndex] = nodeID
 	}
 
+	// We prepare to give some debug output about the size of the graph.
+	totalChannels := 0
+	for _, channels := range graph.Adj {
+		totalChannels += len(channels)
+	}
+
+	// The number of channels is double counted, so divide by two.
+	log.Debugf("initialized simple graph with %d nodes and %d channels",
```

Comment:
nit: log statements (in contrast to error messages) usually start capitalized.

Reply:
Changed it to capital letters.